@xen-orchestra/web-core/lib/components/input/InputLabel.vue

```diff
@@ -0,0 +1,74 @@
+<!-- v2.0 -->
```

Comment:
```suggestion
<!-- v2 -->
```
BTW, i am not seeing the `v2` for this component in the DS. Maybe I'm looking in the wrong place?